### PR TITLE
Add Security and Privacy section for resource timing

### PR DESCRIPTION
**Pull request by quasi-mod**
This PR adds security and privacy section for the resource timing API.
This is still a WIP, as we are currently addressing the reviews by @domenic .

### CODE REVIEW

File: resource-timing-api.md [hunk 1]
```diff
@@ -390,6 +390,37 @@ type is as follows:
 </table>
 
 
+## Security and Privacy Concerns
+
+### Security Concerns
+
+The primary security concern in this proposal is the potential timing
+information leakage of the website, common amongst resource timing information.
+This proposal newly exposes timing information of when the routing occurred, as
```
Comment (yoshisatoyanagisawa) [inline resource-timing-api.md:399]: when cache lookup started?

**Comment by quasi-mod** [comment]
Updated the section.
PTAL.

File: resource-timing-api.md [hunk 2]
```diff
@@ -394,6 +394,36 @@ type is as follows:
...
+### Privacy Concerns
+
+Similar to the security concerns, privacy concern also inherits the same issue
+as the Resource Timing API. It has a potential of fingerprinting.
+While statistical fingerprinting poses a privacy risk, where malicious websites
+can infer a user's browsing history by analyzing resource loading times, the
+Resource Timing API doesn't significantly exacerbate this issue.
+Existing mechanisms, like the load event, already allow for limited cache
+timing measurements. Furthermore, cross-origin restrictions in HTTP Fetch,
+enforced by the Timing-Allow-Origin header, prevent this API from exposing
```
Comment (yoshisatoyanagisawa) [inline resource-timing-api.md:425]: nit but I hope to see "which is" before "enforced".  I just got confused on reading this.

**Review by yoshisatoyanagisawa — APPROVED** [review]
looks good to me other than nit.

File: resource-timing-api.md (hunk 1 above)
Comment (yoshisatoyanagisawa) [reply resource-timing-api.md]: the sentence has been removed, but I think each entry is not so important to discuss in the doc.

File: resource-timing-api.md (hunk 2 above)
Comment (quasi-mod) [reply resource-timing-api.md]: Updated the sentence.

**Review by sisidovski — APPROVED** [review]

**Comment by quasi-mod** [comment]
Could you please merge this PR as I do not have merge permission?

**Pull request merged**